src/pydvl/value/oob/oob.py

```diff
@@ -52,37 +58,53 @@ def compute_data_oob(
     \rightarrow \mathbb{R}
     $$
 
-    T is a score function that represents the goodness of a weak learner $\hat{f}_b$ at the i-th datum $(x_i, y_i)$.
+    T is a score function that represents the goodness of a weak learner
+    $\hat{f}_b$ at the i-th datum $(x_i, y_i)$.
 
-    There is a need to tune n_est and max_samples jointly to ensure all samples are at least 1 time oob, otherwise the result could include a nan value for that datum.
+    `n_est` and `max_samples` must be tuned jointly to ensure that all samples
+    are at least 1 time out-of-bag, otherwise the result could include a NaN
+    value for that datum.
 
     Args:
         u: Utility object with model, data, and scoring function.
         n_est: Number of estimator used in the bagging procedure.
-        max_samples: The fraction of samples to draw to train each base estimator.
-        n_jobs: The number of jobs to run in parallel used in the bagging
-            procedure for both fit and predict.
+        max_samples: The fraction of samples to draw to train each base
+            estimator.
         loss: A function taking as parameters model prediction and corresponding
             data labels(preds, y) and returning an array of point-wise errors.
+        n_jobs: The number of jobs to run in parallel used in the bagging
+            procedure for both fit and predict.
+        seed: Either an instance of a numpy random number generator or a seed
+            for it.
         progress: If True, display a progress bar.
 
     Returns:
         Object with the data values.
     """
+    rng = np.random.default_rng(seed)
+    random_state = np.random.RandomState(rng.bit_generator)
 
     result: ValuationResult[np.int_, np.object_] = ValuationResult.empty(
         algorithm="data_oob", indices=u.data.indices, data_names=u.data.data_names
     )
 
     if is_classifier(u.model):
         bag = BaggingClassifier(
-            u.model, n_estimators=n_est, max_samples=max_samples, n_jobs=n_jobs
+            u.model,
```

Comment:
The utility object is not really used in this method. Perhaps we could define a `BaggingUtility` class? What do you think?

Reply:
I'm not sure, Utility was designed for margin-based methods and here it's being "abused" to conform to the interface. Evaluating this BaggingUtility on subsets could only mean computing the values themselves (i.e. the point-wise losses), which is a vector output and very different semantics from other utilities...

Reply:
You're right. We can leave it as is for now. We will eventually find a better interface.